### PR TITLE
hw16

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**
ready

### CODE REVIEW

**Review by Yanetta — APPROVED** [review]
Good!

File: src/resources/hw16.sql [hunk 1]
```diff
@@ -0,0 +1,315 @@
+/* вывести информацию про заказчиков которые купили продукты с буквой а */
+SELECT
+    *
+FROM
+    customers cs1
+WHERE
+    EXISTS (
```
Comment (SerhiiPasko) [inline src/resources/hw16.sql:7]: Why you use this construction?

Comment (SerhiiPasko) [reply src/resources/hw16.sql:7]: In this case that don't give anithing

File: src/resources/hw16.sql [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+        SELECT
+            *
+        FROM
+            offices
+            INNER JOIN salesreps ON offices.mgr = salesreps.empl_num
+    )
+WHERE
+    target > 600.00;
+
+SELECT
```
Comment (SerhiiPasko) [inline src/resources/hw16.sql:75]: In this case one select in another don't need. Many died(not effective) code

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**